Add canvas or canvas-prebuilt as dependency

We need `canvas` or `canvas-prebuilt` to download images when using `jsdom`.

`canvas-prebuilt` doesn't support node 8 yet and I'm having compiling issues with node 8 and windows:

```
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  Canvas.cc
  CanvasGradient.cc
  CanvasPattern.cc
  CanvasRenderingContext2d.cc
c:\projects\sonar\node_modules\canvas\src\Canvas.h(19): fatal error C1083: Cannot open include file: 'cairo.h': No such file or directory (compiling source file ..\src\CanvasPatt
ern.cc) [C:\Projects\Sonar\node_modules\canvas\build\canvas.vcxproj]
c:\projects\sonar\node_modules\canvas\src\Canvas.h(19): fatal error C1083: Cannot open include file: 'cairo.h': No such file or directory (compiling source file ..\src\CanvasGrad
ient.cc) [C:\Projects\Sonar\node_modules\canvas\build\canvas.vcxproj]
c:\projects\sonar\node_modules\canvas\src\Canvas.h(19): fatal error C1083: Cannot open include file: 'cairo.h': No such file or directory (compiling source file ..\src\CanvasRend
eringContext2d.cc) [C:\Projects\Sonar\node_modules\canvas\build\canvas.vcxproj]
c:\projects\sonar\node_modules\canvas\src\Canvas.h(19): fatal error C1083: Cannot open include file: 'cairo.h': No such file or directory (compiling source file ..\src\Canvas.cc)
 [C:\Projects\Sonar\node_modules\canvas\build\canvas.vcxproj]
```

--- a/.gitignore
+++ b/.gitignore
@@ -3,6 +3,7 @@
 .DS_Store
 /.nyc_output/
 /.vscode/
+/build/
 /coverage/
 /dist/
 /doc/

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,7 +26,7 @@ matrix:
       script:
         - ./.travis/check-links.sh && ./.travis/trigger-site-update.sh
 
-    # 2. Build used for running the tests for Sonar on Ubuntu.
+    # 2. Build used for running the tests for sonar on Ubuntu.
 
     - os: linux
       dist: trusty
@@ -34,22 +34,31 @@ matrix:
         apt:
           sources:
             - google-chrome
+            - ubuntu-toolchain-r-test
           packages:
             - google-chrome-stable
+            - libcairo2-dev
+            - libjpeg8-dev
+            - libpango1.0-dev
+            - libgif-dev
+            - g++-4.9
       before_script:
         - export DISPLAY=:99.0
         - sh -e /etc/init.d/xvfb start
         - sleep 3
+      env:
+        - CXX=g++-4.9
       node_js:
         - node
       script:
         - npm run test-on-travis
 
-    # 3. Build used for running the tests for Sonar on macOS.
+    # 3. Build used for running the tests for sonar on macOS.
 
     - os: osx
       osx_image: xcode8.3
       before_install:
+        - brew install pkg-config cairo libpng jpeg giflib
         - brew cask install google-chrome
       node_js:
         - node

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
   "dependencies": {
     "axe-core": "^2.2.2",
     "browserslist": "^2.1.4",
+    "canvas": "^1.6.5",
     "chalk": "^1.1.3",
     "chrome-remote-interface": "^0.23.0",
     "debug": "^2.6.8",

